public/index.html

```diff
@@ -12,6 +12,8 @@
     <link rel="icon" sizes="16x16" href="<%= BASE_URL %>favicon-16x16.png">
     <link rel="mask-icon" href="<%= BASE_URL %>safari-pinned-tab.svg">
     <link rel="manifest" href="<%= BASE_URL %>site.webmanifest">
+    <link href='//cdn.jsdelivr.net/npm/yasgui-yasqe@2.11.22/dist/yasqe.min.css' rel='stylesheet' type='text/css'/>
+    <link href='//cdn.jsdelivr.net/yasr/2.10.8/yasr.min.css' rel='stylesheet' type='text/css'/>
```

Comment:
You could include these from the npm package instead of loading them from the CDN separately (since you're already installing the package anyway)

Pro's:
- minus one dependency
- you can manage versions more easily (straight within npm)

Should be something like:
```html
<link href='node_modules/yasgui-yasqe/dist/yasqe.min.css' rel='stylesheet' type='text/css'/>
<link href='node_modules/yasgui-yasr/dist/yasr.min.css' rel='stylesheet' type='text/css'/>
```

Reply:
Thanks!
Did not realize they were included in the package.

I think I might even be able to include them in the component and reduce overall file size!

Reply:
Turns out there is an issue with the yasqe package, the css has compile errors.
I've logged an issue.